.github/scripts/update_docker_versions.py

```diff
@@ -67,7 +67,7 @@ def annotate(
     return annotated
 
 
-for filename in (pathlib.Path("data_safe_haven") / "pulumi").glob("**/*.py"):
+for filename in (pathlib.Path("data_safe_haven") / "infrastructure").glob("**/*.py"):
```

Comment:
In the future it would be good to try and collect all the container images and versions together somewhere more visible. Lets not solve that here though.